canvas_modules/common-canvas/src/common-properties/controls/expression/expression-builder/expression-select-field-function.jsx

```diff
@@ -602,44 +612,40 @@ export default class ExpressionSelectFieldOrFunction extends React.Component {
 	}
 
 	render() {
-		const tabContent = [];
 		const fieldAndValueTable = this._makeFieldAndValuesContent();
 		const functionsTable = this._makeFunctionsContent();
 		const fieldsTab = formatMessage(this.reactIntl,
 			MESSAGE_KEYS.EXPRESSION_FIELD_TAB);
 		const functionsTab = formatMessage(this.reactIntl,
 			MESSAGE_KEYS.EXPRESSION_FUNCTIONS_TAB);
+		let content = null;
 
-		tabContent.push(
-			<Tab
-				key={0}
-				id={"expresson-builder-fields-tab"}
-				tabIndex={0}
-				label={fieldsTab}
-				onClick={this.onTabClick.bind(this, 0)}
-			>
-				{fieldAndValueTable}
-			</Tab>
-		);
-
-		tabContent.push(
-			<Tab
-				key={1}
-				id={"expresson-builder-function-tab"}
-				tabIndex={1}
-				label={functionsTab}
-				onClick={this.onTabClick.bind(this, 1)}
-			>
-				{functionsTable}
-			</Tab>
-		);
+		if (this.state.selectedIndex === 0) {
+			content = fieldAndValueTable;
+		} else if (this.state.selectedIndex === 1) {
+			content = functionsTable;
+		}
 
 
 		return (
 			<div className="properties-expression-selection-fieldOrFunction" >
-				<Tabs key={"tab.1"} className="properties-primaryTabs" selected={this.state.selectedTab}>
-					{tabContent}
-				</Tabs>
+				<div className="properties-expression-selection-content-switcher" >
+					<ContentSwitcher onChange={this.onSwitch.bind(this)} selectedIndex={this.state.selectedIndex}>
+						<Switch
+							key={0}
+							name={"expresson-builder-fields-tab"}
+							id={"expresson-builder-fields-tab"}
```

Comment:
id's need to be unique.  Is it required?  If so, these need to be unique since we can have more than 1 common-properties in the dom.

Reply:
The ids are not required, removing them and adding a className instead for testing purposes